src/components/Footer/Contacts.tsx

```diff
@@ -1,28 +1,29 @@
-import React from "react";
+import React, { FC } from "react";
 import Phone from "../../static/svg/icon-phone.svg";
 import Email from "../../static/svg/email.svg";
 import { Icon } from "../Icon";
+import { Link } from "@reach/router";
 
-export const Contacts = () => (
-  <div className="ml-24 mt-8 lg:mt-0 sm:mt-0 mr-16">
-    <div className="pb-4 flex justify-start items-center">
+interface IProps {
+  phoneNumber: number;
+  email: string;
```

Comment:
we don't need this props, this should be hardcoded we don't want to change this or reuse in different place